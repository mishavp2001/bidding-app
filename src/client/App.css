.navbar-right { margin-right: 0px !important}
.log {
  margin: 5px 10px 0 0;
}
.panel-title:after {
  clear: both;
}
.panel-title span{
    float: left;
    margin: 0;
}

.panel-heading .btnDismiss{
  float: right;
  margin: 0;
}

.bids-modal .row {
  border: 1px solid black;
  padding-bottom: 1px;
}

.bids-modal .current-bid {
  background-color: greenyellow;
}

.winning-bid {
  background-color: greenyellow;
  padding: 5px;
}
.hidden {
  display: none;
}

.searchBox input[type=checkbox]{
  margin: 10px 5px;
}

.searchBox label{
  padding: 0 5px;
}
