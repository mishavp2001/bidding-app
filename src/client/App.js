import React from 'react';
import ReactDOM from 'react-dom';
import HomePage from './pages/HomePage';
import UserProjects from './pages/UserProjects';
import Callback from './components/Callback';
import {HashRouter, Route} from 'react-router-dom';
import { requireAuth } from './services/AuthService';
import "./App.css";

const App = () => {
  return (
    <div className="container">
      <HashRouter>
        <Route path="/" component={HomePage}/>
      </HashRouter>
    </div>
  )
}

export default App;
