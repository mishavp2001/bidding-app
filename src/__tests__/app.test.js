jest.mock('../__mocks__/projects-api.js');

import React from 'react'
import renderer from 'react-test-renderer'

import App from '../client/app.js'

// Snapshot for CardComponent React Component
describe('>>>App --- Snapshot',()=>{
    it('+++capturing Snapshot of App', () => {
        const renderedValue =  renderer.create(<App />).toJSON()
        expect(renderedValue).toMatchSnapshot();
    });

});
