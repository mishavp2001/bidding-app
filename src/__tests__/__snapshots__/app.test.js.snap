// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`>>>App --- Snapshot +++capturing Snapshot of App 1`] = `
<div
  className="container"
>
  <div>
    <nav
      className="navbar navbar-default"
    >
      <div
        className="navbar-header"
      >
        <a
          className="navbar-brand"
          href="#/"
          onClick={[Function]}
        >
          Home
        </a>
      </div>
      <ul
        className="nav navbar-nav"
      >
        <li>
          
        </li>
      </ul>
      <ul
        className="nav navbar-nav"
      >
        <li>
          
        </li>
      </ul>
      <ul
        className="nav navbar-nav navbar-right"
      >
        <li>
          <button
            className="btn btn-info log"
            onClick={[Function]}
          >
            Log In
          </button>
        </li>
      </ul>
    </nav>
    <h3
      className="text-center"
    >
      Projects available to bid on
    </h3>
    <div>
      <input
        onChange={[Function]}
        placeholder="Search projects"
        type="text"
      />
      <input
        checked={true}
        onChange={[Function]}
        type="checkbox"
      />
      <label
        htmlFor="showexpired"
      >
        Show expired
      </label>
    </div>
    <hr />
    <div
      className="mainContainer"
    />
    <div
      className="col-sm-12"
    >
      <div
        className="jumbotron text-center"
      />
    </div>
  </div>
</div>
`;
